src/core/utils/cli-config.ts

```diff
@@ -64,14 +75,15 @@ export async function getConfigFileOptions(
 
   logger.silly(`Changed directory to ${configDir}`);
 
+  // In case config name is not provided, this will be the default value from positional argument
   // TODO: comparison here should be a constant, used also as the default context
   // for all commands taking a context argument
-  if (contextOrConfigEntry === `.${path.sep}`) {
+  if (configName === `.${path.sep}`) {
     const hasMultipleConfig = Object.entries(cliConfig.configurations).length > 1;
     if (hasMultipleConfig) {
       // Show as a log not warning because the user may want to use the default config
       logger.log(`Multiple configurations found in "${swaCliConfigFilename}", but none was specified.`);
-      logger.log(`Specify which configuration to use with "swa <command> <configurationName>"\n`);
+      logger.log(`Specify which configuration to use with "swa <command> --config-name <configName>"\n`);
```

Comment:
Suggestion: can we list all found config names so the user can choose from?

Reply:
I think it's best from a DX perspective that we fall back to a default value instead of going interactive every time, as CLI user I would find it annoying. What would be even better though would be a way to set which configuration to use as default, that should be easy to do but more docs/tests/schema to update, I'll see if it can fits before GA